src/Aspire.Dashboard/Components/Pages/Resources.razor

```diff
@@ -42,7 +42,7 @@
             <FluentDataGrid Items="@FilteredResources" ResizableColumns="true" GridTemplateColumns="1fr 2fr 1fr 2fr 3fr 2fr 1fr 1fr" >
                 <ChildContent>
                     <PropertyColumn Property="@(c => c.ResourceType)" Title="Type" Sortable="true" />
-                    <TemplateColumn Title="Name" Sortable="true" SortBy="@_nameSort">
+                    <TemplateColumn Title="Name (Id)" Sortable="true" SortBy="@_nameSort">
```

Comment:
```suggestion
                    <TemplateColumn Title="Name (ID)" Sortable="true" SortBy="@_nameSort">
```

We've been inconsistent about this in the past (and so I'm guessing there's a few places to clean up) but I think the current intent is to make sure to use `ID` when it is displayed to the user.